client/src/app/menu/menu.component.ts

```diff
@@ -47,7 +43,7 @@ export class MenuComponent implements OnDestroy, OnInit {
     }
 
     openUserInformation(): void {
-        const dialogRef = this.dialog.open(ConnectedUserDialogComponent, {
+        this.dialog.open(ConnectedUserDialogComponent, {
```

Comment:
I didn't see that before, but why using a dialog when the user click on his profile ? 
We could just use a [menu](https://material.angular.io/components/menu/overview)

Reply:
I thought that I was going to use it in another component. You are right, I will change that!